content/en/tracing/trace_collection/library_config/nodejs.md

```diff
@@ -75,6 +75,10 @@ Enable automatic injection of trace IDs in logs for supported logging libraries.
 `DD_TRACE_RATE_LIMIT`
 : Percentage of spans to sample as a float between `0` and `1`. Defaults to `100` when `DD_TRACE_SAMPLE_RATE` is set. Otherwise, delegates rate limiting to the Datadog Agent.
 
+`DD_TRACE_SAMPLING_RULES` 
+: **Default**: `[{"sample_rate": 1.0}]` <br>
```

Comment:
This is not correct. The default is an empty rule set and to use a dynamic rate provided by the agent on every flush, which is much lower than 100%.

Reply:
Made some edits. Let me know what you think.